content/en/05-reference/01-admin/04-deploy-options.md

```diff
@@ -7,52 +7,150 @@ redirect_from:
+To provide a custom cluster name when deploying with the [Pixie CLI](/installing-pixie/install-schemes/cli/), use the `--cluster_name` flag:
 
+```bash
+# Deploy Pixie in your K8s cluster and use the provided name for the cluster.
+px deploy --cluster_name=<CLUSTER_NAME>
 ```
-px deploy --data_access=(Full|Restricted)
+
+To provide a custom cluster name when deploying with [Helm](/installing-pixie/install-schemes/helm/), use the `clusterName` field:
+
+```bash
+helm install pixie pixie-operator/pixie-operator-chart --set deployKey=<deploy-key-goes-here> --set clusterName=<CLUSTER_NAME> --namespace pl --create-namespace
 ```
 
-### Helm Install
+## Configuring Pixie's memory usage
+
+When deploying Pixie, you have three options for configuring PEM memory usage:
+
+- [Set PEM pod memory limit](#configuring-pixie's-memory-usage-setting-the-memory-limit)
+- [Set PEM pod memory request](#configuring-pixie's-memory-usage-setting-the-memory-request)
+- [Set data table storage limit](#configuring-pixie's-memory-usage-setting-the-data-table-storage-memory-limit)
+
+For more information about these options, please refer to the [Tuning Memory Usage](/reference/admin/tuning-mem-usage/) page.
+
+### Setting the memory limit
+
+The default memory limit is 2Gi per PEM. The lowest recommended value is [1Gi](/installing-pixie/requirements/#memory) per PEM. 1Gi is not a suitable limit for a cluster with high throughput, but it is suitable for a small cluster with limited resources.
 
-You may specify the desired data access mode using the `dataAccess` field.
+To set Pixie's memory limit when deploying with the [Pixie CLI](/installing-pixie/install-schemes/cli/), use the `--pem_memory_limit` flag:
 
+``` bash
+px deploy --pem_memory_limit=1Gi
+```
+
+To set Pixie's memory limit when deploying with [Helm](/installing-pixie/install-schemes/helm/), use the `pemMemoryLimit` field:
+
+```bash
+helm install pixie pixie-operator/pixie-operator-chart --set deployKey=<deploy-key-goes-here> --namespace pl --create-namespace --set pemMemoryLimit=1Gi
 ```
-helm install pixie pixie-operator/pixie-operator-chart --set deployKey=<deploy-key-goes-here> --set clusterName=<cluster-name> --namespace pl --create-namespace
---dataAccess=(Full|Restricted)
+
+### Setting the memory request
+
+When deploying Pixie, you can specify a different memory [request](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) than limit. This flag is useful for clusters in which the PEM pods are pending because there is not enough memory for them to be scheduled on the node.
+
+<Alert variant="outlined" severity="info">Please note that PEM memory request must be less than or equal to the PEM memory limit.</Alert>
+
+To set Pixie's memory request when deploying with the [Pixie CLI](/installing-pixie/install-schemes/cli/), use the `--pem_memory_request` flag:
```

Comment:
Can you note that by default, the limit and the request will be the same unless they are both specified?